src/main/java/com/mallowigi/icons/TintedIconsComponent.java

```diff
@@ -83,20 +82,17 @@ public void pluginUnloaded(@NotNull final IdeaPluginDescriptor pluginDescriptor,
   }
 
   private void initComponent() {
-    colorPatcher = new TintedColorPatcher();
-    SVGLoader.setColorPatcherProvider(colorPatcher);
+    SVGLoader.setColorPatcherProvider(new TintedColorPatcher(SvgLoaderHacker.collectOtherPatcher()));
 
     // Listen for changes on the settings
     connect.subscribe(LafManagerListener.TOPIC, source -> {
-      SVGLoader.setColorPatcherProvider(null);
-      SVGLoader.setColorPatcherProvider(colorPatcher);
+      SVGLoader.setColorPatcherProvider(new TintedColorPatcher(SvgLoaderHacker.collectOtherPatcher()));
```

Comment:
This captures the new patcher being set, when the LAF changes. If this wasn't done, it would hold on to the old patcher which themes stuff based on the old theme. Which makes things look out of place.